openfecwebapp/templates/partials/candidate/itemized-disbursements.html

```diff
@@ -0,0 +1,68 @@
+{% import 'macros/missing.html' as missing %}
+{% import 'macros/null.html' as null %}
+{% import 'macros/cycle-select.html' as select %}
+
+<section id="section-3" role="tabpanel" aria-hidden="true" aria-labelledby="section-1-heading">
+  <div class="heading--section">
+    <h2>Itemized Disbursements</h2>
+  </div>
+
+  <div class="content__section">
+    {{ select.cycle_select(election_years, cycle, duration=duration, id="cycle-1") }}
```

Comment:
We could try just make the select two-year periods. Though I'm not sure how this will play with the rest of the page and the "Full election" parameter that's set.